### ISSUE TITLE
Config dir does not exist without language pack

### ISSUE THREAD

**Issue by invliD** [saz/puppet-locales#28]
If there is no language pack installed on Ubuntu (tested on 16.04 xenial), the directory `/var/lib/locales/supported.d` does not exist. This makes the locales module fail, because it tries to create the file `local` in that directory without making sure it exists:

```
Error: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /var/lib/locales/supported.d/local20160608-22855-1nxbc2h.lock at 184:/etc/puppet/code/environments/production/modules/locales/manifests/init.pp
Error: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /var/lib/locales/supported.d/local20160608-22855-1nxbc2h.lock at 184:/etc/puppet/code/environments/production/modules/locales/manifests/init.pp
Wrapped exception:
No such file or directory @ dir_s_mkdir - /var/lib/locales/supported.d/local20160608-22855-1nxbc2h.lock
Error: /Stage[main]/Locales/File[/var/lib/locales/supported.d/local]/ensure: change from absent to file failed: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /var/lib/locales/supported.d/local20160608-22855-1nxbc2h.lock at 184:/etc/puppet/code/environments/production/modules/locales/manifests/init.pp
```

**Comment by olevole** [comment]
+1 for issue

**Comment by fnoop** [comment]
+1

**Comment by fnoop** [comment]
Temp workaround:
ensure_packages(["language-pack-en"])
class { "locales":
    default_locale      => "${locale}",
    locales             => [ "${locale}" ],
    require             => Package["language-pack-en"],
}

**Comment by olevole** [comment]
@saz there is any chance to fix it soon ?

**Comment by Yggdrasil** [comment]
I would suggest that the fix proposed in PR #30 is a better solution to this issue. Ubuntu has started to use `/etc/locale.gen`, which IMHO is a better place for a config file than `/var/lib/locales/supported.d/local`.

**Comment by saz** [comment]
This should be resolved in the current master and will be released soon
